bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/EdCompInstBase.java

```diff
@@ -60,26 +67,21 @@ protected void cleanup() {
         }
     }
 
-    // -- Implement the TextBoxWidgetWatcher interface --
-
-
     /**
      * A key was pressed in the given TextBoxWidget.
      */
-    public void textBoxKeyPress(TextBoxWidget tbwe) {
+    public void textBoxKeyPress(final TextBoxWidget tbwe) {
         if (getDataObject() != null) {
             _ignoreChanges = true;
             try {
                 if (tbwe == getPosAngleTextBox()) {
-                    /* The default position angle */
                     final double defaultPositionAngle = 0.0;
                     getDataObject().setPosAngleDegrees(tbwe.getDoubleValue(defaultPositionAngle));
                 } else if (tbwe == getExposureTimeTextBox()) {
-                    double expTime = tbwe.getDoubleValue(getDefaultExposureTime());
-                    if (isForceIntegerExposureTime()) expTime = Math.floor(expTime);
-                    getDataObject().setExposureTime(expTime);
+                    final double expTime    = tbwe.getDoubleValue(getDefaultExposureTime());
+                    final double expTimeAdj = isForceIntegerExposureTime() ? Math.floor(expTime) : expTime;
```

Comment:
It looks like the functionality is in place to allow for this (i.e. there is a protected method `EdCompInstBase.isForceIntegerExposureTime` that returns `false`), but it is never overridden and thus never used.

I'm guessing we can probably just remove all of this.